libfreerdp/locale/timezone.c

```diff
@@ -1523,7 +1523,9 @@ char* freerdp_get_unix_timezone_identifier()
 		return tzid;
 	}
 
-#if defined(__FreeBSD__) || defined(__OpenBSD__)
+#if defined(ANDROID)
+	fp = popen("getprop persist.sys.timezone", "r");
```

Comment:
In case of popen we have to use pclose on Android later on to close the fp correctly.